test/Sema/enum_equatable_hashable.swift

```diff
@@ -12,9 +12,9 @@ var aHash: Int = Foo.A.hashValue
 enum Generic<T> {
   case A, B
 
-  func method() -> Int {
+  static func method() -> Int {
     if A == B { }
-    return A.hashValue
+    return Generic.A.hashValue
```

Comment:
Do we really need `static`?  I'd prefer to stick to the original code as close as possible.